### PR TITLE
apply-personal-number

### PR DESCRIPTION
**Pull request by richardwd**
apply-personal-number

### CODE REVIEW

**Comment by richardwd** [comment]
apply-personal-number

File: apply-personal-number.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# How to apply personal number
+## Step 1:
+##### At first check your map, search a nearest “tax agency”, “tax office” or ”Skattekontoret” (Swedish).
+
+## Step 2：
+##### Bring your Passport, Work permit card; Marriage certificate & legalized translation if not single, Kid's birth certificate & legalized translation if have kids to Skattekontoret. Take a ticket and wait.
+
+## Step 3:
+##### The officer will check your files and you can fill the website on the computer they will find for you. Fill it step by step.
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/1.png)
```
Comment (chuanjin) [inline apply-personal-number.md]: Should use relative path here. Refer [here](https://github.com/Orientalent/guidelines/commit/cfd435fb6741b427ad0f17d37ebe4e738268ea61#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3)

File: apply-personal-number.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/2.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/3.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/4.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/5.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/6.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/7.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/8.png)
+![Aaron Swartz](https://raw.githubusercontent.com/richardwd/guidelines/main/images/pn/9.png)
+
+##### After 1 to 18 months, check the mailbox, and a letter will be sent to you with a personal number and a website. Send the personal number to Ola and cc to CJ by E-mail.
```
Comment (chuanjin) [inline apply-personal-number.md]: Usually the waiting time is 1 to 3 months, check your mailbox regularly, Tax Agency will send you a letter by post with your personal number. When you recieve the letter, send your personal number to admin@orientalent.se to update your infomation in the company.

**Review by chuanjin — CHANGES_REQUESTED** [review]

File: apply-personal-number.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+##### Log in to the website to make an appointment to take pictures on site. 
+
+Link: https://ssc.nemoq.se/booking/booking/index/ssc
+![Aaron Swartz](images/pn/10.png)
+
+Notice: You can book a photoshoot time before you receive the letter. If you haven't received it 2 days before the appointment, you can use the dialog box on the right to change the appointment time. This will save some time.
+
+##### Before taking the photo, you need to pay the card production fee according to the payment method specified in the letter. Approximate 400 kroner.
+
+##### Then go to "Skattekontoret" again to take a picture. After about 2 months, you can get an ID card. Then apply for a bank card with your ID card. Send the bank card number to Ola and cc to CJ by E-mail.
```
Comment (chuanjin) [inline apply-personal-number.md]: Send your bank account number to admin@orientalent.se by E-mail.

**Review by chuanjin — CHANGES_REQUESTED** [review]

**Review by richardwd — COMMENTED** [review]
1.use relative path here
2.Usually the waiting time is 1 to 3 months, check your mailbox regularly, Tax Agency will send you a letter by post with your personal number. When you recieve the letter, send your personal number to admin@orientalent.se to update your infomation in the company.

File: apply-personal-number.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+# How to apply personal number
+## Step 1:
+##### At first check your map, search a nearest “tax agency”, “tax office” or ”Skattekontoret” (Swedish).
+
+## Step 2：
+##### Bring your Passport, Work permit card; Marriage certificate & legalized translation if not single, Kid's birth certificate & legalized translation if have kids to Skattekontoret. Take a ticket and wait.
+
+## Step 3:
+##### The officer will check your files and you can fill the website on the computer they will find for you. Fill it step by step.
+![Aaron Swartz](images/pn/1.png)
```
Comment (chuanjin) [inline apply-personal-number.md]: Remove "Aaron Swartz" and all below

**Review by chuanjin — CHANGES_REQUESTED** [review]

**Review by richardwd — COMMENTED** [review]
Send your bank account number to admin@orientalent.se by E-mail.

**Review by richardwd — COMMENTED** [review]
Remove "Aaron Swartz" and all below

**Review by chuanjin — APPROVED** [review]

**Pull request merged**